jinja2cli/cli.py

```diff
@@ -146,6 +150,23 @@ def _load_xml():
     return xmltodict.parse, xml.parsers.expat.ExpatError, MalformedXML
 
 
+def _load_env():
+    def _parse_env(data):
+        """
+        Parse an envfile format of key=value pairs that are newline separated
+        """
+        dict_ = {}
+        for line in data.splitlines():
+            line = line.strip()
```

Comment:
Also based on what you originally said "dotenv", there are implementation of that that wouldn't cover what you're saying either:
https://www.npmjs.com/package/dotenv#rules for example which is much closer to what I wrote here. I'm matching Docker's behavior.